techniques/index.html

```diff
@@ -65,66 +66,54 @@ <h2>Updated <span class="generate-date"></span></h2>
       		
       <section id="abstract">
          			
-         <h2>Abstract</h2>
+         <h2>概要</h2>
          			
-         <p>This <em>Techniques for WCAG 2.1</em> document provides guidance for web content authors and evaluators on meeting <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines (WCAG) 2.1</a> success criteria. It is part of a series of documents published by the W3C Web Accessibility
-            Initiative (WAI) to support WCAG 2.1. For an introduction to WCAG, supporting technical
-            documents, and educational material, see <a href="https://www.w3.org/WAI/intro/wcag">Web Content Accessibility Guidelines (WCAG) Overview</a>.
+         <p>この文書 <em>WCAG 2.1 達成方法集</em>は、ウェブコンテンツ制作者および評価者向けに <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines (WCAG) 2.1</a> の達成基準を満たすためのガイダンスを提供している。 W3C の Web Accessibility Initiative (WAI) が、WCAG 2.1 を補足するために公開している一連の文書の一つである。WCAG 2.1 のイントロダクション、補足する技術文書、そして教材については <a href="https://www.w3.org/WAI/intro/wcag">Web Content Accessibility Guidelines (WCAG) Overview</a>. を参照のこと。
          </p>
          			
-         <p>Techniques are informative—that means they are not required. The basis for determining
-            conformance to WCAG 2.1 is the success criteria from the <a href="https://www.w3.org/TR/WCAG21/">WCAG 2.1 standard</a>—not the techniques. <strong>For important information about techniques, please see the <a href="https://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques.html">Understanding Techniques for WCAG Success Criteria</a> section of Understanding WCAG 2.1.</strong></p>
+         <p>達成方法は、参考情報である。つまり、達成方法は必須要件ではない。WCAG 2.1 への適合の判断は、<a href="https://www.w3.org/TR/WCAG21/">WCAG 2.1</a> の達成基準に基づくものであり、達成方法ではない。<strong>達成方法に関する重要な情報については、WCAG 2.1 解説書の <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques.html">WCAG 達成基準の達成方法を理解する</a>のセクションを参照のこと。</strong></p>
          			
-         <p><em>Techniques for WCAG 2.1</em> is not intended to be used as a stand-alone document. Instead, it is expected that
-            content authors will usually use <a href="https://www.w3.org/WAI/WCAG21/quickref/">How to Meet WCAG 2.0: A customizable quick reference</a> to read the WCAG success criteria, and follow links from there to specific topics
-            in Understanding WCAG 2.0 and to specific techniques.
-         </p>		
+         <p><em>WCAG 2.1 達成方法集</em>は、独立した文書として利用されることを意図したものではない。コンテンツ制作者が通常は <a href="https://www.w3.org/WAI/WCAG21/quickref/">How to Meet WCAG 2.0: A customizable quick reference</a> を利用して、WCAG の達成基準を読み、そこから WCAG 2.1 解説書にある具体的なトピックや具体的な達成方法へのリンクを辿ることを想定している。
```

Comment:
「2.0」？と思ったのですが、原文でも「How to Meet WCAG 2.0」なんですよね...。でも125行目は「How to Meet WCAG 2.1」になっているので、それに合わせて、ここもしれっと修正する？あるいは、リンク先のタイトルと合わせて「「How to Meet WCAG」（2.0 / 2.1 のバージョン表記なし）にしましょうか？

Reply:
いずれ日本語訳に差し替えますが、 https://waic.jp/docs/WCAG20/quickref/
ここはリンク先にあわせますか。

```suggestion
         <p><em>WCAG 2.1 達成方法集</em>は、独立した文書として利用されることを意図したものではない。コンテンツ制作者が通常は <a href="https://www.w3.org/WAI/WCAG21/quickref/">How to Meet WCAG : A customizable quick reference</a> を利用して、WCAG の達成基準を読み、そこから WCAG 2.1 解説書にある具体的なトピックや具体的な達成方法へのリンクを辿ることを想定している。
```